js/whisper-demo/main.js

```diff
@@ -125,21 +135,27 @@ export async function initWhisper(ort, AutoProcessor, AutoTokenizer, options) {
     return true;
 }
 
+function sleep(ms) {
+    return new Promise(resolve => setTimeout(resolve, ms));
+}
+
 // process audio buffer
-export async function process_audio(audio, starttime, idx, pos, textarea) {
+export async function process_audio(audio, starttime, idx, pos, textarea, progress) {
     if (idx < audio.length) {
         // not done
         try {
             // update progress bar
             progress.style.width = (idx * 100 / audio.length).toFixed(1) + "%";
             progress.textContent = progress.style.width;
+            await sleep(kDelay);
```

Comment:
This sleep will introduce about 100ms additional time for the total process time. I removed it last time.

Reply:
Fixed in https://github.com/Honry/onnxruntime-inference-examples/pull/4/commits/1afe7f53067929939a576b4875b0b6c5040527aa